packages/x-utils/tsconfig.build.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I've created this `tsconfig.build.json` file to be extended by both the `tsconfig.cjs.json` and `tsconfig.esm.json` to extract the common properties.